### DISCUSSION TITLE
i am getting this error with portainer

### DISCUSSION THREAD

**Discussion by ahmadkakarr** [techno-tim/techno-tim.github.io#96] [Questions] [answered]
following these configs https://github.com/techno-tim/techno-tim.github.io/tree/master/reference_files/traefik-portainer-ssl

![image](https://user-images.githubusercontent.com/22844883/143617659-7e08fec3-a4ce-48e4-ab61-f1e669cc2d11.png)

![image](https://user-images.githubusercontent.com/22844883/143617802-74383562-2654-487a-b319-98466ab9fb84.png)

**Comment by ahmadkakarr** [comment]
sorry man, i was trying to modify your config to use portainer-ee/business edition then I got this error, when I changed everything back it worked. sorry
and please can you share what I have to do to run portainer-ee

**Comment by timothystewart6** [comment, marked as answer]
If you're not getting any responses here in the discussions tab, feel free to join our Discord for questions! https://discord.gg/DJKexrJ
